Guides/UPI/vSphere_govc/README.md

```diff
@@ -23,20 +23,64 @@ govc import.ova -ds=<datastore_name> \
-    okd4-master-1 okd4-master-2 okd4-master-3 \
-    okd4-worker-1 okd4-worker-2 okd4-worker-3 \
-    okd4-bootstrap; do
-    govc vm.clone -vm fedora-coreos-31.20200113.3.1-vmware.x86_64 \
-        -ds <datastore_name> -folder <vm_folder> -on=false \
-        -c=4 -m=8192 -net=<network_name> $vm
-
-    govc vm.disk.change -vm $vm -disk.label "Hard disk 1" -size 120G
+#!/bin/bash
+# Title: UPI-vSphere-GenerateVMs
+# Description: This is an example bash script to create the VMs iteratively. Set the values for cluster_name, datastore_name, vm_folder, network_name, master_node_count, and worker_node_count.
+ 
+template_name="fedora-coreos-31.20200113.3.1-vmware.x86_64"
+cluster_name=<cluster_name>
+datastore_name=<datastore_name>
+vm_folder=<folder_path>
+network_name=<network_name>
+master_node_count=<master_node_count>
+worker_node_count=<worker_node_count>
+
+# Create the master nodes
+
+for (( i=1; i<=${master_node_count}; i++ )); do
+        vm="${cluster_name}-master-${i}"
+	govc vm.clone -vm "${template_name}" \
+		-ds "${datastore_name}" \
+		-folder "${vm_folder}" \
+		-on="false" \
+		-c="4" -m="8192" \
+		-net="${network_name}" \
+		$vm
+	govc vm.disk.change -vm $vm -disk.label "Hard disk 1" -size 120G
 done
+
+# Create the worker nodes
+
+for (( i=1; i<=${worker_node_count}; i++ )); do
+        vm="${cluster_name}-worker-${i}"
+        govc vm.clone -vm "${template_name}" \
+                -ds "${datastore_name}" \
+                -folder "${vm_folder}" \
+                -on="false" \
+                -c="4" -m="8192" \
+                -net="${network_name}" \
+                $vm
+	govc vm.disk.change -vm $vm -disk.label "Hard disk 1" -size 120G
+done
+
+
+# Create the bootstrap node
+
+vm="${cluster_name}-bootstrap"
+govc vm.clone -vm "${template_name}" \
+                -ds "${datastore_name}" \
+                -folder "${vm_folder}" \
+                -on="false" \
+                -c="4" -m="8192" \
+                -net="${network_name}" \
+                $vm
+govc vm.disk.change -vm $vm -disk.label "Hard disk 1" -size 120G
+
+
 ```
 
 ### Configure DNS, DHCP and LB
-The installation requires specific configuration of DNS and a load balancer. The requirements are listed in the official OKD documentation: [Creating the user-provisioned infrastructure](https://docs.okd.io/latest/installing/installing_vsphere/installing-vsphere.html#installation-infrastructure-user-infra_installing-vsphere). Example configurations are available at [requirements](/Guides/UPI/vSphere/Requirements)
+The installation requires specific configuration of DNS and a load balancer. The requirements are listed in the official Openshift documentation: [Creating the user-provisioned infrastructure](https://docs.openshift.com/container-platform/4.2/installing/installing_vsphere/installing-vsphere.html#installation-infrastructure-user-infra_installing-vsphere). Example configurations are available at [requirements](/Documentation/UPI/Requirements)
```

Comment:
Lets link okd.io and Guides/UPI/vSphere_govc/Requirements?

Reply:
Fixed.